src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx

```diff
@@ -111,7 +121,7 @@ const MultipleSelectDropDown = ({
           inputProps["aria-label"] = "Types multiple types can be selected";
```

Comment:
I think this should be `inputProps["aria-label"] ="Measure types multiple measure types can be selected"`

Reply:
there is  inputProps["aria-describedby"] = "measure-group-type-helper-text"; prop on this which is not visible here but I think is not required.